### DISCUSSION TITLE
Create report from storage with date range

### DISCUSSION THREAD

**Discussion by jonasengelmann** [allinurl/goaccess#2460] [General]
I like to use the Goaccess storage as it allows me to comply with GDPR regulations in combination with `--anonymize-ip`. I generally remove old access logs. However, I want to create monthly reports. As far as I can see, the only option right now is to filter the access logs and then feed them to Goaccess. Is there also a way to produce monthly reports from storage alone?

**Comment by allinurl** [comment]
If I understand correctly, there's no way right now to extract monthly data out of the persisted db. #117 should let you extract monthly data. Working on it as we speak.

**Reply by jonasengelmann** [reply]
Perfect! #117 looks like it is exactly what I was looking for!
